nginx.conf

```diff
@@ -0,0 +1,71 @@
+# http://wiki.nginx.org/Symfony
+# http://symfony.com/doc/current/cookbook/configuration/web_server_configuration.html
+
+upstream phpfcgi {
+    server phpfpm:9000;
+    # server unix:/var/run/php5-fpm.sock; #for PHP-FPM running on UNIX socket
+}
+
+server {
+    listen 80;
+
+    server_name localhost;
+    root /var/www/symfony2/web;
+
+    # strip app.php/ prefix if it is present
+    rewrite ^/app\.php/?(.*)$ /$1 permanent;
+
+    location / {
+        index app.php;
+        try_files $uri @rewriteapp;
+    }
+
+    location @rewriteapp {
+        rewrite ^(.*)$ /app.php/$1 last;
+    }
+
+    # pass the PHP scripts to FastCGI server from upstream phpfcgi
+    location ~ ^/(app|app_dev|config)\.php(/|$) {
+        fastcgi_pass phpfcgi;
+        fastcgi_split_path_info ^(.+\.php)(/.*)$;
+        include fastcgi_params;
+        fastcgi_param  SCRIPT_FILENAME $document_root$fastcgi_script_name;
+        fastcgi_param  HTTPS off;
+        # Allow access to /app_dev.php & /config.php
+        fastcgi_param  REMOTE_ADDR 127.0.0.1;
```

Comment:
This is only to allow access `/config.php` & `/app_dev.php` without changes in Symfony sources.